lib/client.js

```diff
@@ -2222,6 +2222,123 @@ MatrixClient.prototype.deletePushRule = function(scope, kind, ruleId, callback)
 };
 
 /**
+ * Enable or disable a push notification rule.
+ * @param {string} scope
+ * @param {string} kind
+ * @param {string} ruleId
+ * @param {boolean} enabled
+ * @param {module:client.callback} callback Optional.
+ * @return {module:client.Promise} Resolves: result object
+ * @return {module:http-api.MatrixError} Rejects: with an error response.
+ */
+MatrixClient.prototype.setPushRuleEnabled = function(scope, kind,
+                                                     ruleId, enabled, callback) {
+    var path = utils.encodeUri("/pushrules/" + scope + "/$kind/$ruleId/enabled", {
+        $kind: kind,
+        $ruleId: ruleId
+    });
+    return this._http.authedRequest(
+        callback, "PUT", path, undefined, enabled ? 'true' : 'false'
+    );
+};
+
+/**
+ * Get the room-kind push rule associated with a room.
+ * @param {string} scope "global" or device-specific.
+ * @param {string} roomId the id of the room.
+ * @return {object} the rule or undefined.
+ */
+MatrixClient.prototype.getRoomPushRule = function(scope, roomId) {
+    // There can be only room-kind push rule per room
```

Comment:
Maybe we should throw an exception here or something if the pushrules haven't been fetched yet?